actions/install_package/install_package.sh

```diff
@@ -12,4 +12,4 @@
 
 set -e # Exit with error if any command fails
 
-docker build -t base-${package_name}:latest -f Dockerfile/base-${package_name}.Dockerfile . --build-arg GCC_VERSION=${gcc_version} --build-arg CLANG_VERSION=${clang_version} --build-arg CMAKE_VERSION=${cmake_version} --build-arg UBUNTU_VERSION=${ubuntu_version} --build-arg MADNESS_VERSION=${madness_version} --build-arg CATCH2_VERSION=${catch2_version} --build-arg SPDLOG_VERSION=${spdlog_version} --build-arg CEREAL_VERSION=${cereal_version} --build-arg PARENT_IMAGE_NAME=${parent_image_name}
+docker build -t base-${package_name}:latest -f .github/Dockerfile/base-${package_name}.Dockerfile . --build-arg GCC_VERSION=${gcc_version} --build-arg CLANG_VERSION=${clang_version} --build-arg CMAKE_VERSION=${cmake_version} --build-arg UBUNTU_VERSION=${ubuntu_version} --build-arg MADNESS_VERSION=${madness_version} --build-arg CATCH2_VERSION=${catch2_version} --build-arg SPDLOG_VERSION=${spdlog_version} --build-arg CEREAL_VERSION=${cereal_version} --build-arg PARENT_IMAGE_NAME=${parent_image_name}
```

Comment:
> To avoid the explicit coupling between this file and the downstream repos, I'd suggest making the entire path to the Dockerfile be a parameter instead of just the package name. This allows downstream repos to store the files in different places and to use different names.

You mean pass the variable `${path_to_dockerfile}=.github/Dockerfile/base-parallelzone.Dockerfile` (for example) to the action, and then run the command `docker build -t base-${package_name}:latest -f ${path_to_dockerfile} --build-arg ...`?

Reply:
Yeah.

Reply:
> Yeah.

Dockerfile path parameter just added.